rust/sdk/src/client.rs

```diff
@@ -1089,8 +1089,8 @@ impl<'a> FusedFuture for Subscribe<'a> {
 /// Request builder for monotonic subscriptions.
 ///
 /// Monotonic subscriptions keep track of the highest sort order
-/// ([`LamportTimestamp`](crate::timestamp::LamportTimestamp) and
-/// [`StreamId`](crate::scalars::StreamId)) seen so far, ending the stream with a
+/// ([`LamportTimestamp`](crate::types::LamportTimestamp) and
+/// [`StreamId`](crate::types::StreamId)) seen so far, ending the stream with a
```

Comment:
Note: I'm still not sure about these names or import paths, should we discuss or do you guys like them?